generators/application/index.js

```diff
@@ -81,11 +87,11 @@ class Application extends Generator {
           validate: utils.atLeastOne
         }
       ])
-      components = res.components
+      this.components = res.components
     }
-    const addActions = components.includes('actions')
-    const addEvents = components.includes('events')
-    const addWebAssets = components.includes('webAssets')
+    const addActions = this.components.includes('actions')
```

Comment:
Not your code but, I would rather see `this.components.includes('actions')` used directly instead of defining a single use alias of `addActions`
ex.
```
if (this.components.includes('actions')) { ...
```

Reply:
Was thinking the same thing while working on this, glad someone else mentioned in. Change incoming